ts/masters/modules/data-tools.src.ts

```diff
@@ -0,0 +1,29 @@
+/**
+ * @license Highcharts JS v@product.version@ (@product.date@)
+ * @module highcharts/modules/data-tools
```

Comment:
Is it temporary or is it going to be an official module?

Reply:
Temporary.